tests/integration/debug/cmd_debug_test.go

```diff
@@ -40,14 +40,15 @@ var _ = Describe("odo debug command serial tests", func() {
 		os.Unsetenv("GLOBALODOCONFIG")
 	})
 
-	It("should auto-select a local debug port when the given local port is occupied", func() {
+	It("check that machine output debug information works", func() {
```

Comment:
This test scenario should be in https://github.com/openshift/odo/blob/master/tests/integration/cmd_debug_test.go. This test file is only for test cases which require serial run and the scenario below doesn't seem to require it.